README.md

```diff
@@ -0,0 +1,156 @@
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖  <a name="about-project">Portfolio Mobile </a>
+
+
+**Portfolio Mobile** is a project from the first module in wich we can apply our knowledge in Flexbox and implement the basics items for a website to mobile-divices. 
+
+## 🛠 Built With <a name="built-with">HTML5 & CSS3</a>
+
+### Tech Stack <a name="tech-stack">HTML5</a>
+
+
+<details>
+  <summary>HTML5</summary>
+  <ul>
+    <li><a href="[https://expressjs.com](https://developer.mozilla.org/es/docs/Web/HTML)/">Web-HTML5 </a></li>
+  </ul>
+</details>
+
+### Tech Stack <a name="tech-stack">CSS3</a>
+
+<details>
+  <summary>CSS3</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">Info CSS3</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Set up the linter for fixing the errors and clean code**
+- **Add the structured content using HTML**
+- **Add the styling for the content using CSS**
+--**Work with Flexbox properties in the CSS**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A computer
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+  cd my-folder
+  git clone https://github.com/BiancaMe/Portfolio-Mobile.git
+
+
+### Usage
+
+To run the project, double click on the index to open it in the browser.
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Bianca Lopez**
+
+- GitHub: [@BiancaMe](https://github.com/BiancaMe)
+- Twitter: [@BiancaMeLopez](https://twitter.com/BiancaMeLopez)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/bianca-lopez-55a4a3276/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Display an image **
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project  you can contac me and let me know.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thanks Microverse for give the oportunity  to increase  my knowledge
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Also the MIT license linkis throwing an error 404 when clicked, i would suggest you target the correct name of the MIT license you have in your project root directory and be sure the link is working properly. :+1: :+1: